nelc/ecommerce
# Jlc/palm mig/long lastname

## Description

Migration PR. migration for long fields in lastname and firstname.



## Testing instructions

Open your ecommerce and run migrations. Check there is no problem running them.

[Screencast from 12-02-24 10:49:23.webm](https://github.com/nelc/ecommerce/assets/51926076/911e4868-9214-4bd7-837e-9a8eec1ab50c)
Validate in your mysql that the table ecommerce_user in firstname and lastname have type255 in varchar.
[Screencast from 12-02-24 10:54:40.webm](https://github.com/nelc/ecommerce/assets/51926076/b864a78f-2df0-4e7a-8cf0-f70a51c3a439)
## Other information

Pr related
https://github.com/eduNEXT/ecommerce/pull/40
